src/core/UserIconGroup/UserIconGroup.tsx

```diff
@@ -0,0 +1,146 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import cx from 'classnames';
+import { CommonProps } from '../utils/props';
+
+export type UserIconGroupProps = {
+  /**
+   * UserIconGroup max number of icons unstacked
+   * @default 5
+   */
+  maxIcons?: number;
+  /**
+   * UserIconGroup is group needs to be stacked
```

Comment:
amm, weird english... Just maybe `Is group stacked` or `If true, group will be stacked to take less space.`

Reply:
Changed into not weird English